### PR TITLE
Browser tests

### PR DESCRIPTION
**Pull request by AveryVine**
Started configuring browser tests for Boxwise using [Cypress](https://docs.cypress.io). Tests go in the `boxwise/cypress` folder, and the Cyprus framework can be run using `yarn cypress:open`.

### CODE REVIEW

**Comment by AveryVine** [comment]
Quick note: the tests are incomplete, but shouldn't be breaking and should also save you from having to set up Cyprus yourselves in the future.

**Comment by bfirsh** [comment]
Yesss! Let me test this locally. I'll try and get them running on CI, too. 😄

File: cypress/integration/examples/actions.spec.js [hunk 1]
```diff
@@ -0,0 +1,272 @@
+/// <reference types="Cypress" />
```
Comment (bfirsh) [inline cypress/integration/examples/actions.spec.js]: These examples probably shouldn't be included. :)

**Comment by bfirsh** [comment]
Thanks for removing those. This looks like a great foundation - thanks!

**Pull request merged**